design/ceph/submariner-globalnet.md

```diff
@@ -0,0 +1,96 @@
+# Submariner Globalnet support in Rook
+
+
+## Use case
+As a rook-ceph user, I should be able to setup mirroing across multiple clusters with overlapping networks in order to safegurd my data in case of any disaster.
+
+
+## Background
+[Submariner](https://submariner.io/getting-started/) provides cross-cluster connectivity between two kubernetes clusters when they have different CIDRs. Many a times customers don't have control over the network configuration of the clusters, resulting in clusters with overlapping CIDRs (ServiceCIDR and ClusterCIDR). For such scenarios, Submariner provides another tool called [Globalnet](https://submariner.io/getting-started/architecture/globalnet/). It provides cross-cluster connectivity from pods to remote services using their global IPs.
+
+In order to support kubernetes clusters connected using Submariner-Globalnet, Rook needs to use the global IPs provided by Globalnet instead of the service cluster IP.
+
+
+## Prerequisite
+- The peer clusters should be connected using Submariner.
+- Globalnet controller should be running on both the clusters.
+(Refer Submariner [documentation](https://submariner.io/getting-started/quickstart/) for installation steps on different environments)
+
+## Service of OSD pods
```

Comment:
In addition to the OSD services, mention that the OSD daemon will be aware of the IP with the flags:
```
--public-addr=<externalIP>
--public-bind-addr=<podIP>
```
The latter flag still needs to be implemented by ceph.

Reply:
I've added this point in the `Implementation Details` section below. Let me know that would suffice.